### DISCUSSION TITLE
Permission denied to github-actions[bot]

### DISCUSSION THREAD

**Discussion by mwasifanwar** [community/community#70113] [Actions]
### Select Topic Area

Question

### Body

I keep on running the workflow but giving me the error saying that it can not be run saying permission denied github action bot. Please help me with the issue

**Comment by SaadARazzaq** [comment]
Try checking out this video:
[Permission denied to github-actions[bot]](https://www.youtube.com/watch?v=0pRSFaQYr50&pp=ygUYZ2l0aHViIHNhYWQgYWJkdXIgcmF6emFx)

**Reply by mwasifanwar** [reply]
Thank u so much this video saved my time. I am subscribing to this guy

**Comment by SaadARazzaq** [comment]
Also check out this article I have found: https://stackoverflow.com/questions/72851548/permission-denied-to-github-actionsbot

**Reply by mwasifanwar** [reply]
Amazing

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
